### PR TITLE
Extension redux

### PR DESCRIPTION
**Pull request by broskoski**
extension-example

This goes back to the older style, but with block editing functionality. Going to punt on the keyboard cursors for now, as I can't get the iframe to focus the text field automatically....🙃

### CODE REVIEW

File: src/v2/styles/colors.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const COLORS = {
     neutral: '#fdffdb', // alias editable
     investor: 'rgb(183, 158, 110)',
     supporter: '#00a06d',
+    highlighted: '#fdfdd9',
```
Comment (dzucconi) [inline src/v2/styles/colors.js]: This is suuuuuper close to fdffdb, just use that and/or rename

File: src/extension/src/iframe.css [hunk 2]
```diff
@@ -20,30 +20,31 @@
...
+  padding: 1em 0.5em;
+  -webkit-font-smoothing: antialiased;
+  text-align: center;
+  display: inline-block !important;
   font-size: 1.25em;
   border: dashed 2px #ddd !important;
   margin: 0 auto;
-}
\ No newline at end of file
+}
```
Comment (dzucconi) [inline src/extension/src/iframe.css]: I still think that this should be generated using styled-components.

File: src/extension/src/lib/Pane.js [hunk 3]
```diff
@@ -62,11 +66,16 @@ class Pane {
   }
 
   focusIframe = () => {
-    this.frame.contentWindow.focus()
+    // var frame = this.frame
+    // this.frame.contentWindow.focus()
+    // setTimeout(() => {
+    //   frame.contentWindow.focus()
+    // }, 100)
```
Comment (dzucconi) [inline src/extension/src/lib/Pane.js]: Can delete

**Comment by broskoski** [comment]
Just waiting for @danpia to advise me on what to do with the current empty state, and then this should be ready to go.

**Pull request merged**